### PR TITLE
Release 20200622

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist 

- [x] Correct Markdown and formatting
- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] All content reviewed

-----
[View rendered README.md](https://github.com/publiccodenet/about/blob/release-20200622/README.md)
[View rendered activities/codebase-stewardship/community-assets.md](https://github.com/publiccodenet/about/blob/release-20200622/activities/codebase-stewardship/community-assets.md)
[View rendered activities/codebase-stewardship/index.md](https://github.com/publiccodenet/about/blob/release-20200622/activities/codebase-stewardship/index.md)
[View rendered activities/procurement/index.md](https://github.com/publiccodenet/about/blob/release-20200622/activities/procurement/index.md)
[View rendered activities/procurement/principles.md](https://github.com/publiccodenet/about/blob/release-20200622/activities/procurement/principles.md)
[View rendered activities/procurement/process.md](https://github.com/publiccodenet/about/blob/release-20200622/activities/procurement/process.md)
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/release-20200622/contributor-guides/for-staff.md)

### CODE REVIEW

File: activities/codebase-stewardship/community-assets.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+This is a checklist of the most important community assets needed to put an existing codebase on its path to incubation, roughly organized by when they're needed.
+
+## Assumptions
+
+We assume that the chronological goals during incubation are to:
+
+* make sure the codebase is deliberately designed and documented to be reusable
+* create everything needed to enable and support a contributing community
+* focus on collecting evidence and marketing to attract more replicators and contributors
+* improve an active codebase based on performance metrics, user research or new feature requests
```
Comment (bvhme) [inline activities/codebase-stewardship/community-assets.md:16]: @ElenaFdR, do you think these _need_ trailling punctuation?

Comment (bvhme) [reply activities/codebase-stewardship/community-assets.md:16]: I've not checked that last box, however if you think no change is necessary then I'd say we're good to go.

Comment (ElenaFdR) [reply activities/codebase-stewardship/community-assets.md:16]: I do not think these need trailing punctuation.

**Comment by bvhme** [comment]
I've also updated develop.

**Pull request merged**